### PR TITLE
newdevice: AXIS Audio Appliance

### PR DESCRIPTION
**Pull request by EnzoZafra**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #6829

### CODE REVIEW

File: tests/snmpsim/axisaudio.snmprec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|  ; AXIS P8221; Network IO Audio Module; 5.10.3; Jan 29 2016 10:47; 174; 1;
```
Comment (laf) [inline tests/snmpsim/axisaudio.snmprec]: Is this both the complete sysDescr and sysObjectId?

File: includes/polling/os/axisaudio.inc.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 Lorenzo Zafra
+ * @author     Lorenzo Zafra<zafra@ualberta.ca>
+ */
+
+// .1.3.6.1.2.1.1.1.0 = STRING:  ; AXIS P8221; Network IO Audio Module; 5.10.3; Jan 29 2016 10:47; 174; 1;
```
Comment (laf) [inline includes/polling/os/axisaudio.inc.php]: You can't get rid of this comment.

File: includes/polling/os/axisaudio.inc.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * $axisaudio.inc.php
```
Comment (laf) [inline includes/polling/os/axisaudio.inc.php]: Can drop the $ from the file name.

File: tests/snmpsim/axisaudio.snmprec (hunk 1 above)
Comment (EnzoZafra) [inline tests/snmpsim/axisaudio.snmprec]: Yup

**Pull request merged**